src/components/Nav/Nav.jsx

```diff
@@ -1,33 +1,90 @@
-import React from "react";
-
+import React, { useState } from "react";
 import * as s from "./Nav.style";
+import logoSrc from "@Assets/images/full-white-2.png";
+import logoSrcBlack from "@Assets/images/full-black-2.png";
 
 const Nav = (props) => {
-    const { barsolid } = props;
-    return (
-        <div className="Nav-Container">
-            <s.NavList>
-                <s.NavListItem>
-                    <s.Link $barsolid={barsolid} to={""} end>Home</s.Link>
-                </s.NavListItem>
-                <s.NavListItem>
-                    <s.Link $barsolid={barsolid} to={"/products"}>Products</s.Link>
-                </s.NavListItem>
-                <s.NavListItem>
-                    <s.Link $barsolid={barsolid} to={"/company-timeline"}>Company Timeline</s.Link>
-                </s.NavListItem>
-                <s.NavListItem>
-                    <s.Link $barsolid={barsolid} to={"/student-timeline"}>Student Timeline</s.Link>
-                </s.NavListItem>
-                <s.NavListItem>
-                    <s.Link $barsolid={barsolid} to={"/contact-us"}>Contact Us</s.Link>
-                </s.NavListItem>
-                <s.NavListItem>
-                    <s.ExternalLink $barsolid={barsolid} href={"https://gorgeous-dragon-4d0ca6.netlify.app/"} target="_blank">Login</s.ExternalLink>
-                </s.NavListItem>
-            </s.NavList>
-        </div>
-    );
+  const [visible, setVisible] = useState(false);
```

Comment:
just for convention, consider changing the state variables to isVisible and setIsVisible

Reply:
oopsss accidentally pushed to main https://github.com/Gopher-Industries/Company-Website/commit/35bc0e98c7dcc681e9a3fc1239afa8de38a851b2